bridge/e2e/src/user-menu.e2e-spec.ts

```diff
@@ -0,0 +1,49 @@
+import { AppPage } from './app.po';
+import { browser, by, element, logging, protractor } from 'protractor';
+
+import { takeScreenshot } from './utils';
+
+// run with: ng e2e --dev-server-target= --base-url=http://localhost:3000/
+describe('User Menu', () => {
+  let page: AppPage;
+
+  beforeEach(() => {
+    page = new AppPage();
+  });
+
+  it('should open the user menu and copy api token', async () => {
+    await page.navigateTo();
+
+    await element(
+      by.xpath('//*[@uitestid="keptn-nav-userMenu"]')
+    ).click();
+    takeScreenshot('user-menu-open.png');
+
+    await element(
+      by.xpath('//*[@uitestid="keptn-nav-copyKeptnApiToken"]/ktb-copy-to-clipboard/div/div[2]/dt-copy-to-clipboard/button')
+    ).click();
+
+    await browser.executeScript(() => {
+      let el = document.createElement('input');
+      el.setAttribute('id', 'apiTokenTempInput');
+      document.getElementsByTagName('body')[0].appendChild(el);
+    });
+
+    const apiTokenTempInput = element(by.id('apiTokenTempInput'));
+    await apiTokenTempInput.sendKeys(protractor.Key.chord(protractor.Key.CONTROL, 'v'));
+
+    const val = await apiTokenTempInput.getAttribute('value');
```

Comment:
I did a short research, and apparently this way of "simulating" copy/paste does not work in chrome headless mode (and it does not work on my local ubuntu either).

I suggest to not use the copy/paste feature, but to "reveal" the token and take it from the actual input field :)
![image](https://user-images.githubusercontent.com/56065213/120504950-d6d19d80-c3c4-11eb-8e76-ba6e4c52ab70.png)

Reply:
Done :)